src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -173,7 +173,7 @@ namespace AppInstaller::CLI::Workflow
         info << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstaller << std::endl;
         if (installer)
         {
-            info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerType << ' ' << Manifest::InstallerTypeToString(installer->InstallerType) << std::endl;
+            info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerType << ' ' << Manifest::InstallerTypeToString(installer->BaseInstallerType) << std::endl;

```

Comment:
I think that this should be something like:
```
Effective (Base)
```
When effective and base are not the same.  The important part is the effective type, but also showing the base is useful.

Reply:
Changed to suggested. 
 
If effective and base are not the same, will show both "Effective (Base)". Added tests to verify output